docs/usage/self-hosted-configuration.md

```diff
@@ -333,8 +329,6 @@ JSON files will be stored inside the `cacheDir` beside the existing file-based p
 
 Warning: this is an experimental feature and may be modified or removed in a future non-major release.
 
-TODO: Check if this feature is still present in Renovate code.
```

Comment:
Is `repositoryCache` still present in the Renovate code?

We warn that: "this is an experimental feature and may be modified or removed in a future non-major release."

Reply:
Yes, it's still present

Reply:
Okay, we can keep the `Warning: ....` text then. 😄